app/Application.php

```diff
@@ -300,6 +265,8 @@ protected function authorize()
             }
         } while ($user['password'] !== $password);
 
+        $this->historyService->setUser($user['username']);
```

Comment:
it would be better to keep full user data

```suggestion
        $this->historyService->setUser($user);
```